README.md

```diff
@@ -45,3 +45,11 @@ They are:
 	- Example: `[{"address":"1e2345...",rank: 4, "githubHandle":"user-1"},{"address":"1e4532...",rank: 3, "githubHandle":"user-2"},{"address":"1e8335...",rank: 6}]`
 - `github-handles`: All the fellows handles separated by commas
 	- Example: `user-1,user-2,user-3`
+
+## Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics). See the following image for an example:
```

Comment:
Idea to consider:

You can direct the user to a link like this:
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics/decode/0x1c010407676974687562123c67697468756220757365726e616d653e0000000000000000

Then instruct the user to switch to the `Submission` - then you have the transaction ready, only the github username have to be changed.